### PR TITLE
V1.5.0 notebooks

### PR DESCRIPTION
**Pull request by w-k-jones**
Add new script to run notebooks, based off of @lettlini s actions
Re-generate notebooks for v1.5.0 release

* [x] Have you followed our guidelines in CONTRIBUTING.md? 
* [x] Have you self-reviewed your code and corrected any misspellings? 
* [x] Have you written documentation that is easy to understand?
* [x] Have you written descriptive commit messages? 
* [x] Have you added NumPy docstrings for newly added functions? 
* [x] Have you formatted your code using black? 
* [x] If you have introduced a new functionality, have you added adequate unit tests?
* [x] Have all tests passed in your local clone? 
* [x] If you have introduced a new functionality, have you added an example notebook?
* [x] Have you kept your pull request small and limited so that it is easy to review? 
* [x] Have the newest changes from this branch been merged?

### CODE REVIEW

**Review by freemansw1 — APPROVED** [review]
See my one comment. I'm not interested in holding up v1.5.0 for it, so I'm happy to merge now.

File: run_notebooks.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+find . -type f -name '*.ipynb' -not -path "*/.ipynb_checkpoints/*" > nbfiles.txt
```
[the remark this answers is not in the document]
Comment (w-k-jones) [reply run_notebooks.sh:2]: Have added this now

**Comment by freemansw1** [comment]
@w-k-jones happy for you to override and merge; also happy to wait on @JuliaKukulies if she wants to review (or if you want her to review)

**Comment by w-k-jones** [comment]
Will merge straight away as this doesn't change anything in the tobac package itself

**Comment by JuliaKukulies** [comment]
Note to @fsenf and me to include this in the dev guide #281

**Pull request merged**